### DISCUSSION TITLE
Cannot get got-to-definition to work with PnP + VSCode -- help?

### DISCUSSION THREAD

**Discussion by jeff-wishnie** [yarnpkg/berry#5143] [Q&A] [answered]
Hello,

I've performed the setup:
- install and enable ZipFS extension
- install yarn SDK
- Select the workspace/sdk version of TSC

And still I get this when I go to a definition:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/84994260/208208342-d37e33e7-75b3-4213-a509-f39bc153f1b2.png">

Additional info:
os: macOS
vscode: 1.74.1
other relevant extentions:
- Volar in Takeover Mode
- builtin TypeScript services disabled

**Comment by merceyz** [comment, marked as answer]
Duplicate of https://github.com/yarnpkg/berry/issues/3123

**Reply by jeff-wishnie** [reply]
Thanks. So no current workaround? Is this potentially fixable within Volar, or just a limitation of what can be done?
